xt/lib/Pherkin/Extension/LedgerSMB.pm

```diff
@@ -162,10 +164,19 @@ sub create_user {
     return $user;
 }
 
+sub check_database {
+    my ($self,$db) = @_;
+    my $returnstring = `psql -c "SELECT 1 FROM pg_database WHERE datname='$db'"`;
+    return grep { /^ +1$/ } split("\n", $returnstring);
+}
 
 sub create_template {
     my ($self) = @_;
 
+    if ($self->last_scenario_stash->{one_db} && $self->check_database($self->template_db_name)) {
```

Comment:
The template must be created, once. If PSQL reports not found, let's create it. If we have one_db and the template is there, forget it.